presto-postgresql/src/main/java/com/facebook/presto/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -83,6 +113,20 @@ protected String toSqlType(Type type)
         return super.toSqlType(type);
     }
 
+    @Override
+    public Optional<ReadMapping> toPrestoType(ConnectorSession session, JdbcTypeHandle typeHandle)
+    {
+        switch (typeHandle.getJdbcType()) {
```

Comment:
I'm not sure this operation is safe. `CLOB` can be any text and doesn't need to be json right?

Reply:
Yeah, I CLOB changes are not really required in here. I have taken that out